README.md

```diff
@@ -124,6 +130,7 @@ All icons used are the 16px variant available, which fit perfectly in normal tex
 | [`UnwatchRepository.svg`][OcticonsUnwatchRepository] | ![OcticonsUnwatchRepository] |       |
 | [`WatchRepository.svg`][OcticonsWatchRepository]     | ![OcticonsWatchRepository]   |       |
 | [`Wiki.svg`][OcticonsWiki]                           | ![OcticonsWiki]              |       |
+| [`People.svg`][OcticonsPeople]                           | ![OcticonsPeople]              |       |
```

Comment:
Move this under `ForkedRepository.svg` to keep it alphabetical.
Also, remove 4 spaces from the second `|`